aws/resource_aws_emr_managed_scaling_policy_test.go

```diff
@@ -0,0 +1,107 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/emr"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccAwsEmrManagedScalingPolicy_basic(t *testing.T) {
+	resourceName := "aws_emr_managed_scaling_policy.testpolicy"
+	rInt := acctest.RandInt()
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSEmrManagedScalingPolicyDestroy,
+
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSEmrManagedScalingPolicy_basic(rInt),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAWSEmrManagedScalingPolicyExists(resourceName),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccAWSEmrManagedScalingPolicy_basic(r int) string {
+	return fmt.Sprintf(testAccAWSEmrInstanceGroupBase+`
```

Comment:
This configuration is currently failing the acceptance test:

```
=== CONT  TestAccAwsEmrManagedScalingPolicy_basic
TestAccAwsEmrManagedScalingPolicy_basic: resource_aws_emr_managed_scaling_policy_test.go:18: Step 1/2 error: terraform failed: exit status 1
stderr:
Error: error putting EMR Managed Scaling Policy: ValidationException: Managed scaling is not available for the account for this release.
  status code: 400, request id: e93be3ca-c2c1-4e7e-8160-94154349391a
```

In general we try to avoid re-using configurations across resource files. My suggestion would be to copy the configuration contained in `testAccAWSEmrInstanceGroupBase` into this file and adjust it (including the release version) until this passes. 👍

Reply:
acceptance test fixed.
```
make testacc TESTARGS='-run=TestAccAwsEmrManagedScalingPolicy_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAwsEmrManagedScalingPolicy_basic -timeout 120m
=== RUN   TestAccAwsEmrManagedScalingPolicy_basic
=== PAUSE TestAccAwsEmrManagedScalingPolicy_basic
=== CONT  TestAccAwsEmrManagedScalingPolicy_basic
--- PASS: TestAccAwsEmrManagedScalingPolicy_basic (465.80s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	465.863s
```